frontend/src/components/App/RouteSwitcher.tsx

```diff
@@ -44,8 +44,10 @@ export default function RouteSwitcher() {
             requiresAuth={!route.noAuthRequired}
             requiresCluster={getRouteUseClusterURL(route)}
             exact={!!route.exact}
-            children={<RouteComponent route={route} />}
-            key={index}
+            clusters={useClustersConf()}
```

Comment:
This is a hook, so be careful if it's being used conditionally here (hooks must be used conditionally). Maybe bring it earlier in the component's code.

Reply:
fixed, thanks